src/librustdoc/clean/utils.rs

```diff
@@ -504,10 +507,44 @@ pub(super) fn render_macro_arms<'a>(
 /// as part of an item declaration.
 pub(super) fn render_macro_matcher(tcx: TyCtxt<'_>, matcher: &TokenTree) -> String {
```

Comment:
Can you move this function and all its support functions (like the keyword-spacing one) to a new module, say `clean/render_macro_matchers.rs`?

Reply:
Done.